__tests__/app.test.js

```diff
@@ -37,13 +37,47 @@ describe("news api test suite", () => {
   });
 });
 
+describe("GET /api/articles/:article_id test suite", () => {
+  test("returns an article object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
```

Comment:
Because we're returning a specific article for id 1 - we can be specific with the expected content returned. I.e. your test should test that reponse.body.article.article_id = 1 and not just any "number" - the same for all of the other expects.